### ISSUE TITLE
bones-generate succeeds, but build fails due to non-generated files

### ISSUE THREAD

**Issue by jgosmann** [nengo/nengo-bones#41]
Currently, running `bones-generate` gives me the following output
```
No config entry detected for docs_conf_py, skipping
No config entry detected for setup_cfg, skipping
No config entry detected for setup_py, skipping
No config entry detected for contributing_rst, skipping
No config entry detected for contributors_rst, skipping
No config entry detected for license_rst, skipping
No config entry detected for manifest_in, skipping
```
and a 0 exit code which looks like the missing config entries are just warnings that can be safely ignored. However, on Travis CI Nengo bones then complains that these files are either not existing or not generated with Nengo bones and fails the build ([see this build for example](https://travis-ci.org/nengo/nengo-spa/jobs/543616379)).

To provide a better developer experience according to the fail fast principle, `generate-bones` should indicate that these are hard errors (non-zero exit code and message that those config section are required) or the Travis CI build should continue anyways.

**Comment by drasmuss** [comment]
This is just due to a bug, having non-versioned files shouldn't be an error. Will fix!

**Comment by drasmuss** [comment]
Fixed in #45
